deploy/helm/sumologic/conf/metrics/otelcol/config.yaml

```diff
@@ -31,279 +31,9 @@ extensions:
 {{ end }}
   pprof: {}
 exporters:
-  ## Configuration for Sumo Logic Exporter
```

Comment:
Is the plan to have the otel metrics collector share its config with otel metadata?

Reply:
That's what I want to do, yes.